api/service/networkinfo/service.go

```diff
@@ -42,6 +42,9 @@ var (
 const (
 	waitInRetry       = 2 * time.Second
 	connectionTimeout = 3 * time.Minute
+
+	// register to bootnode every ticker
+	dhtTicker = 12 * time.Hour
```

Comment:
How about every 6 hours. For 12 hours, if the first announce fails, then the DHT will be cleaned too.

Reply:
okay. changed.